packages/autorest.python/autorest/codegen/__init__.py

```diff
@@ -26,7 +26,7 @@ def _default_pprint(package_name: str) -> str:
 class OptionsRetriever:
     OPTIONS_TO_DEFAULT = {
         "azure-arm": False,
-        "unbranded": False,
+        "flavor": "azure",  # need to default to azure in shared code so we don't break swagger generation
```

Comment:
i have `autorest.python` code defaulting `flavor` to `azure` so we don't break exixting swagger generations. However, by default the tsp emitters pass `flavor=undefined` to the autorest generator

Reply:
Perfect, thanks for the clarification